Do not display exception on graceful exit case

This is a dupe of @davidfowl's #298, but I cannot reopen it. I ran `dotnet counters monitor -p <pid>` and saw the following output after trying to gracefully quit:

```
Press p to pause, r to resume, q to quit.
    Status: Running

[System.Runtime]
    % Time in GC since last GC (%)                                  11
    Allocation Rate (B / 1 sec)                            352,338,192
    CPU Usage (%)                                                   97    [ERROR] System.Exception: Read past end of stream.Exception Count (Count / 1 sec)                                  0
   at FastSerialization.IOStreamStreamReader.Fill(Int32 minimum)     2.921
   at FastSerialization.MemoryStreamReader.ReadByte()           10,670
   at FastSerialization.Deserializer.ReadObject()                    0
   at Microsoft.Diagnostics.Tracing.EventPipeEventSource.Process(),760
   at Microsoft.Diagnostics.Tools.Counters.CounterMonitor.<>c__DisplayClass21_0.<Start>b__1() in /_/src/Tools/dotnet-counters/CounterMonitor.cs:line 337                                    24                                48.548s 13:57:35
C:\dev❯
    Gen 2 Size (B)                                                  24
    IL Bytes Jitted (B)                                         25,108
    LOH Size (B)                                         2,592,103,176
    Monitor Lock Contention Count (Count / 1 sec)                   43
    Number of Active Timers                                          0
    Number of Assemblies Loaded                                      8
    Number of Methods Jitted                                       217
    POH (Pinned Object Heap) Size (B)                               24
    ThreadPool Completed Work Item Count (Count / 1 sec)         1,346
    ThreadPool Queue Length                                          1
    ThreadPool Thread Count                                         20
    Working Set (MB)                                            10,706
[MyApp]
    invocations-rate (Count / 1 sec)                         7,988,806
```

```
> dotnet counters --version
5.0.160202+5734230e3ee516339a4b0e4729def135027aa255
```

It looks like this was originally fixed by #300 but regressed by #1635.

--- a/src/Tools/dotnet-counters/CounterMonitor.cs
+++ b/src/Tools/dotnet-counters/CounterMonitor.cs
@@ -356,7 +356,7 @@ namespace Microsoft.Diagnostics.Tools.Counters
                 }
                 catch (Exception ex)
                 {
-                    Console.WriteLine($"[ERROR] {ex.ToString()}");
+                    Debug.WriteLine($"[ERROR] {ex.ToString()}");
                 }
                 finally
                 {

